CI is failing

It looks like there is a problem in some of the dependencies needing an update.

Below is the output from the failed run:

```
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/command/sdist.py", line 251, in add_defaults
          self._add_defaults_ext()
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/command/sdist.py", line 336, in _add_defaults_ext
          self.filelist.extend(build_ext.get_source_files())
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "<string>", line 201, in get_source_files
        File "/tmp/pip-build-env-cdrc14e2/overlay/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 107, in __getattr__
          raise AttributeError(attr)
      AttributeError: cython_sources
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

Notice:  A new release of pip is available: 23.2.1 -> 23.3.1
Notice:  To update, run: pip install --upgrade pip
Error: Process completed with exit code 1.
```

Complete CI output: 
[errors.txt](https://github.com/spdx/spdx-spec/files/13230044/errors.txt)

--- a/requirements.txt
+++ b/requirements.txt
@@ -3,3 +3,4 @@ json-schema-for-humans==0.47
 mkdocs-pdf-export-plugin==0.5.10
 mike==1.1.2
 shacl2code==0.0.9
+PyYAML>=6.0.1

